src/components/TextWithHighlight/TextWithHighlight.tsx

```diff
@@ -96,17 +98,19 @@ const TextWithHighlight: React.FC<TextWithHighlightProps> = forwardRef(
 
     useLayoutEffect(() => {
       if (componentRef.current) {
-        componentRef.current.style.setProperty("--heading-clamp-lines", linesToClamp);
+        componentRef.current.style.setProperty("--heading-clamp-lines", linesToClamp.toString());
```

Comment:
just wonder, why we needed to add the .toString()?

Reply:
setProperty methods just accepts string as an argument